scripts/macpython-install-python.sh

```diff
@@ -408,15 +351,15 @@ function make_workon_venv {
 if test "$(arch)" == "arm64"; then
   echo "we are arm"
   PLAT=arm64
-  for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11; do
+  for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11 $LATEST_3p12 $LATEST_3p13; do
```

Comment:
We do not want 3.12 or 3.13. This adds extra install time and disk space on the the build systems. Our builds for Python 3.11 work with later versions.

Reply:
Fixed. added comment to indicate the reason for the limited set of supported versions.